Source/WebKit/NetworkProcess/Cookies/WebCookieManager.h

```diff
@@ -84,6 +84,10 @@ class WebCookieManager : public NetworkProcessSupplement, public IPC::MessageRec
     void platformSetHTTPCookieAcceptPolicy(PAL::SessionID, WebCore::HTTPCookieAcceptPolicy, CompletionHandler<void()>&&);
     void getHTTPCookieAcceptPolicy(PAL::SessionID, CompletionHandler<void(WebCore::HTTPCookieAcceptPolicy)>&&);
 
+#if USE(SOUP)
+    void setCookieJar(PAL::SessionID, const Vector<WebCore::Cookie>&, CompletionHandler<void()>&&);
```

Comment:
This should be renamed too, we are not setting a new jar, but replacing the cookies in a existing jar.